libiocage/lib/Config/ConfigFile.py

```diff
@@ -29,11 +29,11 @@
 
 class ConfigFile(libiocage.lib.Config.Prototype.Prototype):
 
-    _file: str = None
+    _file: str
 
     def __init__(
         self,
-        file: str=None,
+        file: str,
```

Comment:
`file` is explicitly allowed to be None. There are situations where the file is unknown. For example when a new jail gets created before name assignment.